### DISCUSSION TITLE
Option for timer to continue after tab is closed

### DISCUSSION THREAD

**Discussion by tilden** [proginosko/LeechBlockNG#351] [Ideas]
I'm a big fan of the HackerNews `noprocrast` mode, which works much like LeechBlock in that you specify an allow timer and an interval for when that timer resets.

However, while LeechBlock's timer pauses if a tab is closed, HackerNews's timer starts the moment you open a new HN tab, and continues until it reaches zero regardless of whether or not you are actively viewing HackerNews. After the allow time is reached, HN is entirely blocked until the reset time is reached. I prefer this behavior, as it discourages repeatedly viewing a site for short periods of time.

Any chance LeechBlock could have an option like this, for timers to start immediately after loading a site and continue even if that tab was closed? Thanks!

**Comment by proginosko** [comment]
Thanks for the suggestion. Right now, I can't see a way to implement this that wouldn't require major changes to the code. But I will give it some thought.
